### PR TITLE
Carbonblack and Auth0 Collector Node v14 Update

### PR DESCRIPTION
**Pull request by imkariya**
### Solution Description
How it was fixed? 

#### Carbonblack :

- Node Version 14 updated in sam template
- Package Version Updated in package Json
- Profile changes updated in run-sm script
- Missing parameters added in env json

#### Auth0 :

- Added env.json.tmpl file
- Added event_poll.json file
- Added run-sm.sh file
- Added sam-template.yaml file
- Package Version Updated in package Json

### CODE REVIEW

**Review by kkuzmin — CHANGES_REQUESTED** [review]
the change contains both Auth0 and Carbon black changes. Is this intended?

**Comment by imkariya** [comment]
> the change contains both Auth0 and Carbon black changes. Is this intended?

Yes, We can proceed With Both.

File: collectors/auth0/local/events/event_poll.json [hunk 1]
```diff
@@ -0,0 +1,20 @@
+{
+    "Records": [
+        {
+            "messageId": "f67a45fc-ab43-4df1-9f2f-585bb3043192",
+            "receiptHandle": "AQEB78LmKaaZ+uj8DVnc/O2zQcAe+qKSi3ZGTSBFssIRSmpwwzUEi2vhPTaIBnhOoh0aoRxVjWdoXO3ZloINfMxmcjycP3KC0WXwcWokoOc3iMCdqhYg0NcOhQW1X0ixc79C9/5/XF1xGd79vLhFL7KvRjjiT4sOaSxlAv6v2fJ5eDETnp7CRa5pocCF4EO2su0M4/TnlLreGfsY+C+/tH+r19AM+d3Jt5dbNMrKMWRRZ7/PTjczkIM7U38AHPuusuBz9uzA5yMQGOMI8FPXfqgcafEN17JqKuNSd/l54v1+s9rBQSzL/MH9wZ0XpZditcpe5pTc+dGuRHOXbFK2A0YUQCvRq1Ed8tfr68uELTQK5jWHH/zPnEMWsTyco9VuNCKr",
+            "body": "{\n  \"priv_collector_state\": {\n    \"since\": \"2019-11-19T10:00:00Z\",\n    \"until\": \"2019-11-19T10:30:00Z\",\n  \"stream\": \"SearchAlerts\",\n  \"nextPage\":null \n }\n}",
```
Comment (kkuzmin) [inline collectors/auth0/local/events/event_poll.json:6]: This is wrong auth0 state.
Please find an example [here](https://github.com/alertlogic/paws-collector/blob/master/collectors/auth0/auth0_collector.js#L39-L42)

**Review by kkuzmin — CHANGES_REQUESTED** [review]

File: collectors/auth0/local/events/event_poll.json (hunk 1 above)
Comment (imkariya) [reply collectors/auth0/local/events/event_poll.json:6]: Okay Thanks, I will make changes and test accordingly.

Comment (imkariya) [reply collectors/auth0/local/events/event_poll.json]: I have Corrected auth0 state,
[click here](https://github.com/alertlogic/paws-collector/blob/7ec1d7c5fd2a2a6580957ef3ecbe54f3a4dac6f2/collectors/auth0/local/events/event_poll.json#L6)

**Review by kkuzmin — APPROVED** [review]

**Pull request merged**